EarlySamples/InitialProject/InitialProject/Assets/Scripts/Test2/TestPlayerInputManager.cs

```diff
@@ -0,0 +1,107 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.InputSystem;
+
+public class TestPlayerInputManager : MonoBehaviour
+{
+    private static TestPlayerInputManager instance; // the singleton instance of DummyTestGamemanager
+    public static TestPlayerInputManager Instance { get { return instance; } } // The get property of the singleton
+
+    public InputActionAsset testActions;
+    InputActionMap keyboardActionMap;
```

Comment:
Does this really have anything to do with a keyboard? Does not appear so in the code.

Reply:
Naming changed to playerMovementMap.